index.js

```diff
@@ -3,16 +3,12 @@
 const chalk = require( 'chalk' )
 const figlet = require( 'figlet' )
 const package = require( './package.json' )
-const path = require( 'path' )
-const isExecGlobal = require( './lib/is-exec-global.js' )
 
 const args = process.argv
-const isGlobal = isExecGlobal( args[ 1 ] )
-const exec = isGlobal ? path.parse( args[ 1 ] ).name : path.relative( process.cwd(), args[ 1 ] )
+const exec = 'smk'
 
 const command = ( args[ 2 ] || '' ).toLowerCase()
 const opt = require( 'minimist' )( args.slice( 3 ) )
-opt.isGlobal = isGlobal
 opt.exec = exec
```

Comment:
Do you know the purpose of this code that was removed? Did it all exist solely to get the name of the executable?

Reply:
It looks like "is global" functionality is only used for the value of "exec". I'm not sure why it was assigned to opt though. I was curious myself and did a git blame on the lib file removed and saw it was added in commit 950537a with the message "detect if smk is installed globally". I don't see other usages of isGlobal.

Reply:
Did you try a 'local' install of the smk-cli to make sure it still works? I'm pretty sure it will, but better safe than sorry!

Reply:
Yes, I've tested changes locally, confirming "smk", "smk help", "smk create" and "smk edit" continue to work as expected. I've also repeated these edits on the installed node module in my Windows VM and confirmed the same functionality there. I'd be happy to demo it since I've done the setup already.

Reply:
No that's great. I'm surprised creating a new npm package (with npm init command) and incorporating the smk-cli even works! Thanks!